java/ql/lib/semmle/code/java/dataflow/ExternalFlow.qll

```diff
@@ -300,6 +300,10 @@ private predicate summaryModelCsv(string row) {
       "java.net;URI;false;toAsciiString;;;Argument[-1];ReturnValue;taint",
       "java.io;File;false;toURI;;;Argument[-1];ReturnValue;taint",
       "java.io;File;false;toPath;;;Argument[-1];ReturnValue;taint",
+      "java.io;File;true;getAbsoluteFile;;;Argument[-1];ReturnValue;taint",
```

Comment:
For consistency with the two lines above should this be changed to not consider subtypes (`false`), though `java.io.File` can be subclassed (even though it is unlikely that many subclasses exist), so maybe the lines above should be changed (`false` → `true`)?

Reply:
The class can, and does (rarely) get extended. Solid example of it happening here:

https://github.com/gradle/gradle/blob/4f26967b87f2bdc66927348d8c04bb26c1238967/subprojects/internal-testing/src/main/groovy/org/gradle/test/fixtures/file/TestFile.java#L72

Reply:
Updated to change them all to `true`